corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less

```diff
@@ -42,6 +42,13 @@
   }
 }
 
+#sidebar-region {
+  background: transparent;
+  table {
```

Comment:
Seeing the table rule made me look at split screen on staging and wonder if this could be achieved by removing the `table-striped` class?

Reply:
Sort of, but then the small distinction there is between cells gets lost ("Name" is actually highlighted here, if it looks half a shade lighter):
<img width="314" alt="Screenshot 2023-07-25 at 2 02 14 PM" src="https://github.com/dimagi/commcare-hq/assets/100609770/9bebbc7a-34de-4c67-8676-37a934973e5a">

I personally think it looks a little nicer, but I also think losing that distinction is actually a bigger UI change than this PR is intended for.